reputation/reputation.go

```diff
@@ -0,0 +1,413 @@
+package reputation
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/google/uuid"
+	// import of sqlite3 for side effects
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// RepRow is the Data type for Rows in Reputation table
+type RepRow struct {
+	name               string
+	timestamp          string
+	uptime             int
+	auditSuccess       int
+	auditFail          int
+	latency            int
+	amountOfDataStored int
+	falseClaims        int
+	shardsModified     int
+}
+
+// starts a sqlite3 database from the file path parameter
+func startDB(filePath string) *sql.DB {
+	db, err := sql.Open("sqlite3", filePath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return db
+}
+
+// creates a table in sqlite3 based on the create table string parameter
+func createTable(createStmt string, db *sql.DB) {
+	_, err := db.Exec(createStmt)
+	if err != nil {
+		log.Printf("%q: %s\n", err, createStmt)
+		return
+	}
+}
+
+// creates a reputation row struct with a name field base on the name parameter
+func createNamedRow(seed int, name string) RepRow {
+	return RepRow{
+		name,
+		"",
+		seed,
+		seed,
+		seed,
+		seed,
+		seed,
+		seed,
+		0,
+	}
+}
+
+// create a slice of reputation row with random data with the number of rows based on the max row parameter
+func createRandRows(maxRows int) []RepRow {
+	var res []RepRow
+
+	for i := 0; i < maxRows+1; i++ {
+		uid := uuid.New()
+
+		row := createNamedRow(i, uid.String())
+
+		res = append(res, row)
```

Comment:
nit -- 
```Go
res = append(res, createNamedRow(i, uuid.New().String())
```

Reply:
I think the way James has it is more clear to me though